In [1]:
# IMPORTANT: SOME KAGGLE DATA SOURCES ARE PRIVATE
# RUN THIS CELL IN ORDER TO IMPORT YOUR KAGGLE DATA SOURCES.
# import kagglehub
# kagglehub.login()


In [2]:
# IMPORTANT: RUN THIS CELL IN ORDER TO IMPORT YOUR KAGGLE DATA SOURCES,
# THEN FEEL FREE TO DELETE THIS CELL.
# NOTE: THIS NOTEBOOK ENVIRONMENT DIFFERS FROM KAGGLE'S PYTHON
# ENVIRONMENT SO THERE MAY BE MISSING LIBRARIES USED BY YOUR
# NOTEBOOK.

# titanic_path = kagglehub.competition_download('titanic')

# print('Data source import complete.')


UnauthenticatedError: User is not authenticated

# Titanic competition with TensorFlow Decision Forests

This notebook will take you through the steps needed to train a baseline Gradient Boosted Trees Model using TensorFlow Decision Forests and creating a submission on the Titanic competition.

This notebook shows:

1. How to do some basic pre-processing. For example, the passenger names will be tokenized, and ticket names will be splitted in parts.
1. How to train a Gradient Boosted Trees (GBT) with default parameters
1. How to train a GBT with improved default parameters
1. How to tune the parameters of a GBTs
1. How to train and ensemble many GBTs

# Imports dependencies

In [1]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf

print(f"Found TF-DF {tfdf.__version__}")

2025-05-29 11:46:14.218576: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.


Found TF-DF 1.9.0


# Load dataset

In [2]:
train_df = pd.read_csv("titanic/train.csv")
serving_df = pd.read_csv("titanic/test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

# Prepare dataset

We will apply the following transformations on the dataset.

1. Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].
2. Extract any prefix in the ticket. For example ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [3]:
def preprocess(df):
    df = df.copy()

    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])

    def ticket_number(x):
        return x.split(" ")[-1]

    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])

    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)
    return df

preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

Let's keep the list of the input features of the model. Notably, we don't want to train our model on the "PassengerId" and "Ticket" features.

In [4]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


# Convert Pandas dataset to TensorFlow Dataset

In [5]:
def tokenize_names(features, labels=None):
    """Divide the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

# Train model with default parameters

### Train model

First, we are training a GradientBoostedTreesModel model with the default parameters.

In [6]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 25-05-29 11:47:10.0036 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:47:10.0075 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:47:10.0075 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:47:13.7548 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpdhyd6apd/model/ with prefix 822d6e39d37e42b9
[INFO 25-05-29 11:47:13.7733 PDT quick_scorer_extended.cc:911] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 25-05-29 11:47:13.7749 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 25-05-29 11:47:13.7749 PDT kernel.cc:1061] Use fast generic engine


Accuracy: 0.804347813129425 Loss:0.8922085165977478


# Train model with improved default parameters

Now you'll use some specific parameters when creating the GBT model

In [7]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"

    #num_trees=2000,

    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,

    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",

    #num_trees=1000,
    #tuner=tuner

    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,

)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 25-05-29 11:47:31.1879 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:47:31.1880 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:47:31.1880 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:47:31.8083 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpegb4nuhc/model/ with prefix 1f93ed96ff624502
[INFO 25-05-29 11:47:31.8187 PDT decision_forest.cc:734] Model loaded with 54 root(s), 2824 node(s), and 10 input feature(s).
[INFO 25-05-29 11:47:31.8187 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesGeneric" built
[INFO 25-05-29 11:47:31.8188 PDT kernel.cc:1061] Use fast generic engine


Accuracy: 0.77173912525177 Loss:1.0136744976043701


Let's look at the model and you can also notice the information about variable importance that the model figured out

In [8]:
model.summary()

Model: "gradient_boosted_trees_model_1"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
Total params: 1 (1.00 Byte)
Trainable params: 0 (0.00 Byte)
Non-trainable params: 1 (1.00 Byte)
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.431313 ################
    2.           "Age"  0.373986 ############
    3.          "Fare"  0.250863 ####
    4.          "Name"  0.225272 ###
    5.   "Ticket_item"  0.182111 
    6.      "Embarked"  0.181387 
    7. "Ticket_number"  0.180897 
    8.        "Pclass"  0.178279 
    9.         "Parch"  0.175300 
   10.         "SibSp"  0.172167 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 39.00000

# Make predictions

In [ ]:
# def prediction_to_kaggle_format(model, threshold=0.5):
#     proba_survive = model.predict(serving_ds, verbose=0)[:,0]
#     return pd.DataFrame({
#         "PassengerId": serving_df["PassengerId"],
#         "Survived": (proba_survive >= threshold).astype(int)
#     })

# def make_submission(kaggle_predictions):
#     path="/kaggle/working/submission.csv"
#     kaggle_predictions.to_csv(path, index=False)
#     print(f"Submission exported to {path}")

# kaggle_predictions = prediction_to_kaggle_format(model)
# make_submission(kaggle_predictions)
# !head /kaggle/working/submission.csv

# Training a model with hyperparameter tunning

Hyper-parameter tuning is enabled by specifying the tuner constructor argument of the model. The tuner object contains all the configuration of the tuner (search space, optimizer, trial and objective).


In [9]:
tuner = tfdf.tuner.RandomSearch(num_trials=1000)
tuner.choice("min_examples", [2, 5, 7, 10])
tuner.choice("categorical_algorithm", ["CART", "RANDOM"])

local_search_space = tuner.choice("growing_strategy", ["LOCAL"])
local_search_space.choice("max_depth", [3, 4, 5, 6, 8])

global_search_space = tuner.choice("growing_strategy", ["BEST_FIRST_GLOBAL"], merge=True)
global_search_space.choice("max_num_nodes", [16, 32, 64, 128, 256])

#tuner.choice("use_hessian_gain", [True, False])
tuner.choice("shrinkage", [0.02, 0.05, 0.10, 0.15])
tuner.choice("num_candidate_attributes_ratio", [0.2, 0.5, 0.9, 1.0])


tuner.choice("split_axis", ["AXIS_ALIGNED"])
oblique_space = tuner.choice("split_axis", ["SPARSE_OBLIQUE"], merge=True)
oblique_space.choice("sparse_oblique_normalization",
                     ["NONE", "STANDARD_DEVIATION", "MIN_MAX"])
oblique_space.choice("sparse_oblique_weights", ["BINARY", "CONTINUOUS"])
oblique_space.choice("sparse_oblique_num_projections_exponent", [1.0, 1.5])

# Tune the model. Notice the `tuner=tuner`.
tuned_model = tfdf.keras.GradientBoostedTreesModel(tuner=tuner)
tuned_model.fit(train_ds, verbose=0)

tuned_self_evaluation = tuned_model.make_inspector().evaluation()
print(f"Accuracy: {tuned_self_evaluation.accuracy} Loss:{tuned_self_evaluation.loss}")

Use /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp0odci14c as temporary training directory


[WARNING 25-05-29 11:49:28.7941 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:49:28.7941 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:49:28.7942 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:52:56.6647 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp0odci14c/model/ with prefix c62b6a1b82724e4a
[INFO 25-05-29 11:52:56.6783 PDT decision_forest.cc:734] Model loaded with 19 root(s), 1843 node(s), and 12 input feature(s).
[INFO 25-05-29 11:52:56.6784 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesGeneric" built
[INFO 25-05-29 11:52:56.6785 PDT kernel.cc:1061] Use fast generic engine


Accuracy: 0.8767123222351074 Loss:0.6639896035194397


In the last line in the cell above, you can see the accuracy is higher than previously with default parameters and parameters set by hand.

This is the main idea behing hyperparameter tuning.

For more information you can follow this tutorial: [Automated hyper-parameter tuning](https://www.tensorflow.org/decision_forests/tutorials/automatic_tuning_colab)

# Making an ensemble

Here you'll create 100 models with different seeds and combine their results

This approach removes a little bit the random aspects related to creating ML models

In the GBT creation is used the `honest` parameter. It will use different training examples to infer the structure and the leaf values. This regularization technique trades examples for bias estimates.

In [10]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)

    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

#make_submission(kaggle_predictions)

i:0


[WARNING 25-05-29 11:54:10.6607 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:10.6608 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:10.6608 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:11.1246 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpozkk89r8/model/ with prefix 7c3d7de0cdfb4428
[INFO 25-05-29 11:54:11.1278 PDT quick_scorer_extended.cc:911] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 25-05-29 11:54:11.1281 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 25-05-29 11:54:11.1282 PDT kernel.cc:1061] Use fast generic engine


i:1


[WARNING 25-05-29 11:54:11.5727 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:11.5727 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:11.5727 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 25-05-29 11:54:12.0700 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp_iwq59su/model/ with prefix 5d901b9939b54d9e
[INFO 25-05-29 11:54:12.0782 PDT kernel.cc:1061] Use fast generic engine


i:2


[WARNING 25-05-29 11:54:12.4720 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:12.4720 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:12.4721 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 25-05-29 11:54:12.9882 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpl8u929n5/model/ with prefix 37114d60818d4a66
[INFO 25-05-29 11:54:12.9967 PDT kernel.cc:1061] Use fast generic engine


i:3


[WARNING 25-05-29 11:54:13.4187 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:13.4187 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:13.4187 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:13.8658 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp5_s2cle8/model/ with prefix b777c04ca8f74d41
[INFO 25-05-29 11:54:13.8721 PDT kernel.cc:1061] Use fast generic engine


i:4


[WARNING 25-05-29 11:54:14.2623 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:14.2623 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:14.2623 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:14.6415 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp48a_0po7/model/ with prefix 46425c05c96443ae
[INFO 25-05-29 11:54:14.6453 PDT kernel.cc:1061] Use fast generic engine


i:5


[WARNING 25-05-29 11:54:15.0119 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:15.0119 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:15.0119 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:15.3996 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp3g85gs2w/model/ with prefix 8b728ef7a0bd47c5
[INFO 25-05-29 11:54:15.4039 PDT kernel.cc:1061] Use fast generic engine


i:6


[WARNING 25-05-29 11:54:15.7905 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:15.7905 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:15.7905 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:16.2398 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpv061vyil/model/ with prefix 7c159e69c98f4aea
[INFO 25-05-29 11:54:16.2470 PDT kernel.cc:1061] Use fast generic engine


i:7


[WARNING 25-05-29 11:54:16.6282 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:16.6283 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:16.6283 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:17.0389 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpmrq_2ypm/model/ with prefix 6d96f9f1fd1b47be
[INFO 25-05-29 11:54:17.0438 PDT kernel.cc:1061] Use fast generic engine


i:8


[WARNING 25-05-29 11:54:17.4212 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:17.4212 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:17.4213 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:17.9273 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpy0v2z7ra/model/ with prefix 182f8b3582334a59
[INFO 25-05-29 11:54:17.9365 PDT kernel.cc:1061] Use fast generic engine


i:9


[WARNING 25-05-29 11:54:18.3032 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:18.3032 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:18.3032 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:18.8379 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpi46fe86v/model/ with prefix 584184aec2424596
[INFO 25-05-29 11:54:18.8487 PDT kernel.cc:1061] Use fast generic engine


i:10


[WARNING 25-05-29 11:54:19.2728 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:19.2729 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:19.2729 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:19.6804 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp39q3wj6a/model/ with prefix 4ea1d9770f304c5e
[INFO 25-05-29 11:54:19.6857 PDT kernel.cc:1061] Use fast generic engine


i:11


[WARNING 25-05-29 11:54:20.6071 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:20.6072 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:20.6072 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:21.1892 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp46jlbikl/model/ with prefix de04081774a34b52
[INFO 25-05-29 11:54:21.2005 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 25-05-29 11:54:21.2005 PDT kernel.cc:1061] Use fast generic engine


i:12


[WARNING 25-05-29 11:54:21.6340 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:21.6341 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:21.6341 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:22.0933 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpob83kmmj/model/ with prefix b4ab1b3eb235444b
[INFO 25-05-29 11:54:22.1012 PDT kernel.cc:1061] Use fast generic engine


i:13


[WARNING 25-05-29 11:54:22.4958 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:22.4959 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:22.4959 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:23.2651 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpkmnnl_r_/model/ with prefix 4a0ed20851ec4916
[INFO 25-05-29 11:54:23.2871 PDT kernel.cc:1061] Use fast generic engine


i:14


[WARNING 25-05-29 11:54:23.6619 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:23.6619 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:23.6620 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:24.0986 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp6qc9v600/model/ with prefix 05b5ff58ca654c23
[INFO 25-05-29 11:54:24.1048 PDT kernel.cc:1061] Use fast generic engine


i:15


[WARNING 25-05-29 11:54:24.4942 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:24.4943 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:24.4943 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:24.8904 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp9k6lxerd/model/ with prefix 739239bf4d024357
[INFO 25-05-29 11:54:24.8948 PDT kernel.cc:1061] Use fast generic engine


i:16


[WARNING 25-05-29 11:54:25.2543 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:25.2544 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:25.2544 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:25.7257 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpk8xg26m9/model/ with prefix 325b8e131bee4d03
[INFO 25-05-29 11:54:25.7339 PDT kernel.cc:1061] Use fast generic engine


i:17


[WARNING 25-05-29 11:54:26.1561 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:26.1562 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:26.1562 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:26.5313 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp2ui33_1s/model/ with prefix 3a7a9e9e9bca4648
[INFO 25-05-29 11:54:26.5360 PDT kernel.cc:1061] Use fast generic engine


i:18


[WARNING 25-05-29 11:54:26.8955 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:26.8955 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:26.8955 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:27.3974 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmprzb0zxir/model/ with prefix 2081c1e4defa4d54
[INFO 25-05-29 11:54:27.4067 PDT kernel.cc:1061] Use fast generic engine


i:19


[WARNING 25-05-29 11:54:27.8362 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:27.8363 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:27.8363 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:28.3317 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpzpq105r9/model/ with prefix ac7570f38c2849a9
[INFO 25-05-29 11:54:28.3398 PDT kernel.cc:1061] Use fast generic engine


i:20


[WARNING 25-05-29 11:54:28.7362 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:28.7362 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:28.7363 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:29.1353 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpvpxrqzu5/model/ with prefix 819e0eaa06934496
[INFO 25-05-29 11:54:29.1400 PDT kernel.cc:1061] Use fast generic engine


i:21


[WARNING 25-05-29 11:54:29.5079 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:29.5079 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:29.5079 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:29.8959 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpin823g1w/model/ with prefix 05dfe676fb2b46f8
[INFO 25-05-29 11:54:29.9004 PDT kernel.cc:1061] Use fast generic engine


i:22


[WARNING 25-05-29 11:54:30.2538 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:30.2538 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:30.2538 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:30.6844 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmphktcbxu6/model/ with prefix e6463628783e4550
[INFO 25-05-29 11:54:30.6914 PDT kernel.cc:1061] Use fast generic engine


i:23


[WARNING 25-05-29 11:54:31.0513 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:31.0513 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:31.0513 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:31.4338 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpweoab0mw/model/ with prefix b6260ed4292b47c1
[INFO 25-05-29 11:54:31.4394 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 25-05-29 11:54:31.4394 PDT kernel.cc:1061] Use fast generic engine


i:24


[WARNING 25-05-29 11:54:31.7953 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:31.7953 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:31.7953 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:32.1755 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpvpurxicf/model/ with prefix 4775ad9f4c304536
[INFO 25-05-29 11:54:32.1807 PDT kernel.cc:1061] Use fast generic engine


i:25


[WARNING 25-05-29 11:54:32.5364 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:32.5365 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:32.5365 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:32.9661 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp8cwe144a/model/ with prefix 9e0aab3c219a4cc8
[INFO 25-05-29 11:54:32.9729 PDT kernel.cc:1061] Use fast generic engine


i:26


[WARNING 25-05-29 11:54:33.3387 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:33.3387 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:33.3387 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:33.7100 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpemwbbqxv/model/ with prefix d507ac5f047f448a
[INFO 25-05-29 11:54:33.7139 PDT kernel.cc:1061] Use fast generic engine


i:27


[WARNING 25-05-29 11:54:34.0763 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:34.0764 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:34.0764 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:34.5711 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpzuhs9dxi/model/ with prefix 164df9d326d44758
[INFO 25-05-29 11:54:34.5791 PDT kernel.cc:1061] Use fast generic engine


i:28


[WARNING 25-05-29 11:54:34.9346 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:34.9347 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:34.9347 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:35.2701 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp8qwrlwov/model/ with prefix ccf6566421c94a03
[INFO 25-05-29 11:54:35.2730 PDT kernel.cc:1061] Use fast generic engine


i:29


[WARNING 25-05-29 11:54:35.6364 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:35.6364 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:35.6364 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:36.0142 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpnq1tdnrq/model/ with prefix a717a36b2f6a4f1c
[INFO 25-05-29 11:54:36.0186 PDT kernel.cc:1061] Use fast generic engine


i:30


[WARNING 25-05-29 11:54:36.6908 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:36.6909 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:36.6909 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:37.2094 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpw94h1pbd/model/ with prefix 26b35e889deb4b2d
[INFO 25-05-29 11:54:37.2175 PDT kernel.cc:1061] Use fast generic engine


i:31


[WARNING 25-05-29 11:54:37.6133 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:37.6133 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:37.6133 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:38.2307 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp7eqjnyra/model/ with prefix 233cc229f79c41c4
[INFO 25-05-29 11:54:38.2434 PDT kernel.cc:1061] Use fast generic engine


i:32


[WARNING 25-05-29 11:54:38.6324 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:38.6324 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:38.6325 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:39.2092 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpzvsgo2sc/model/ with prefix bc4c7f9e9b234504
[INFO 25-05-29 11:54:39.2223 PDT kernel.cc:1061] Use fast generic engine


i:33


[WARNING 25-05-29 11:54:39.5933 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:39.5933 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:39.5934 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:40.0226 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpgbbc5k1g/model/ with prefix b84f2b6a038049ba
[INFO 25-05-29 11:54:40.0291 PDT kernel.cc:1061] Use fast generic engine


i:34


[WARNING 25-05-29 11:54:40.3898 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:40.3898 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:40.3898 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:40.8110 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpuqeelj3i/model/ with prefix a769fe6d388e48f0
[INFO 25-05-29 11:54:40.8162 PDT kernel.cc:1061] Use fast generic engine


i:35


[WARNING 25-05-29 11:54:41.2142 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:41.2142 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:41.2142 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:41.6579 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpnnbywd6r/model/ with prefix 9550d2e87ae74e5b
[INFO 25-05-29 11:54:41.6643 PDT quick_scorer_extended.cc:911] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 25-05-29 11:54:41.6647 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 25-05-29 11:54:41.6648 PDT kernel.cc:1061] Use fast generic engine


i:36


[WARNING 25-05-29 11:54:42.1467 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:42.1468 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:42.1468 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:42.5544 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpiete18do/model/ with prefix 9a73bc8a0b294949
[INFO 25-05-29 11:54:42.5592 PDT kernel.cc:1061] Use fast generic engine


i:37


[WARNING 25-05-29 11:54:42.9679 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:42.9679 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:42.9680 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:43.3710 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpgcnxit60/model/ with prefix b72bf611cc1d4b3a
[INFO 25-05-29 11:54:43.3752 PDT kernel.cc:1061] Use fast generic engine


i:38


[WARNING 25-05-29 11:54:43.7894 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:43.7894 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:43.7894 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:44.2601 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpr2nup7di/model/ with prefix cd8c985c8b55461b
[INFO 25-05-29 11:54:44.2677 PDT kernel.cc:1061] Use fast generic engine


i:39


[WARNING 25-05-29 11:54:44.6360 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:44.6361 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:44.6361 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:45.3523 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmphmue02zq/model/ with prefix f94a762631ac42e6
[INFO 25-05-29 11:54:45.3659 PDT kernel.cc:1061] Use fast generic engine


i:40


[WARNING 25-05-29 11:54:45.7736 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:45.7737 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:45.7737 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:46.1988 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpv4qohlh3/model/ with prefix cf057e0873124187
[INFO 25-05-29 11:54:46.2041 PDT kernel.cc:1061] Use fast generic engine


i:41


[WARNING 25-05-29 11:54:46.6598 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:46.6599 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:46.6599 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:47.1942 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpmjikne8f/model/ with prefix 9a4fa53104274566
[INFO 25-05-29 11:54:47.2026 PDT kernel.cc:1061] Use fast generic engine


i:42


[WARNING 25-05-29 11:54:47.5705 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:47.5706 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:47.5706 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:47.9973 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpzlmr2qrm/model/ with prefix acfb29e3643c4f60
[INFO 25-05-29 11:54:48.0027 PDT kernel.cc:1061] Use fast generic engine


i:43


[WARNING 25-05-29 11:54:48.4037 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:48.4038 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:48.4038 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:48.9362 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp1yf5suny/model/ with prefix 60ba03bc2ff44c8e
[INFO 25-05-29 11:54:48.9473 PDT kernel.cc:1061] Use fast generic engine


i:44


[WARNING 25-05-29 11:54:49.4094 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:49.4095 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:49.4095 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:49.8180 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp5z49wgex/model/ with prefix 4f344ccad6c04710
[INFO 25-05-29 11:54:49.8237 PDT kernel.cc:1061] Use fast generic engine


i:45


[WARNING 25-05-29 11:54:50.1908 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:50.1909 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:50.1909 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:50.5278 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmphj5qf0a1/model/ with prefix e10ce5c63c3145eb
[INFO 25-05-29 11:54:50.5304 PDT kernel.cc:1061] Use fast generic engine


i:46


[WARNING 25-05-29 11:54:50.8839 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:50.8840 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:50.8840 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:51.3253 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpvsk0vk17/model/ with prefix ac26da0e41d84761
[INFO 25-05-29 11:54:51.3323 PDT kernel.cc:1061] Use fast generic engine


i:47


[WARNING 25-05-29 11:54:51.6842 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:51.6842 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:51.6842 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:52.1100 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpvsdjs5e7/model/ with prefix 9cd77cf66191426d
[INFO 25-05-29 11:54:52.1150 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 25-05-29 11:54:52.1150 PDT kernel.cc:1061] Use fast generic engine


i:48


[WARNING 25-05-29 11:54:52.4706 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:52.4706 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:52.4706 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:52.8433 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpfp1cr_ay/model/ with prefix 03b13179ddc242e4
[INFO 25-05-29 11:54:52.8471 PDT kernel.cc:1061] Use fast generic engine


i:49


[WARNING 25-05-29 11:54:53.2121 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:53.2122 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:53.2122 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:53.6315 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpp5gz4r2q/model/ with prefix 01e93187356e4aaf
[INFO 25-05-29 11:54:53.6377 PDT kernel.cc:1061] Use fast generic engine


i:50


[WARNING 25-05-29 11:54:53.9966 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:53.9966 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:53.9966 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:54.5502 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpx8go8f7g/model/ with prefix cddb18e6bd5c4343
[INFO 25-05-29 11:54:54.5616 PDT kernel.cc:1061] Use fast generic engine


i:51


[WARNING 25-05-29 11:54:54.9325 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:54.9325 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:54.9325 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:55.9701 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpxi_ua9wt/model/ with prefix a14bd09872364449
[INFO 25-05-29 11:54:55.9832 PDT kernel.cc:1061] Use fast generic engine


i:52


[WARNING 25-05-29 11:54:56.3621 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:56.3621 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:56.3621 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:56.7964 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp2cwn_dxk/model/ with prefix 7950bdeb139b44f3
[INFO 25-05-29 11:54:56.8031 PDT kernel.cc:1061] Use fast generic engine


i:53


[WARNING 25-05-29 11:54:57.2055 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:57.2055 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:57.2055 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:57.6167 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmplpozbhxx/model/ with prefix d1670c6912a04a52
[INFO 25-05-29 11:54:57.6224 PDT kernel.cc:1061] Use fast generic engine


i:54


[WARNING 25-05-29 11:54:57.9841 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:57.9842 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:57.9842 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:58.4698 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpvpmnt8yw/model/ with prefix 158245bea233451e
[INFO 25-05-29 11:54:58.4765 PDT kernel.cc:1061] Use fast generic engine


i:55


[WARNING 25-05-29 11:54:58.8690 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:58.8690 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:58.8691 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:54:59.4109 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpi5cyy3qa/model/ with prefix 920442f4ebaf4a39
[INFO 25-05-29 11:54:59.4198 PDT kernel.cc:1061] Use fast generic engine


i:56


[WARNING 25-05-29 11:54:59.8430 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:59.8431 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:54:59.8431 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:00.2739 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp0b9tft4u/model/ with prefix f9a7b819f49d40c4
[INFO 25-05-29 11:55:00.2792 PDT kernel.cc:1061] Use fast generic engine


i:57


[WARNING 25-05-29 11:55:00.6566 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:00.6566 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:00.6566 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:01.0840 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp2r0lg9b4/model/ with prefix b096913b5c8a48a4
[INFO 25-05-29 11:55:01.0894 PDT kernel.cc:1061] Use fast generic engine


i:58


[WARNING 25-05-29 11:55:01.4754 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:01.4755 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:01.4755 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:02.0463 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp4u814zq_/model/ with prefix a6ce0ba9fffb4303
[INFO 25-05-29 11:55:02.0579 PDT kernel.cc:1061] Use fast generic engine


i:59


[WARNING 25-05-29 11:55:02.4305 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:02.4306 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:02.4306 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:02.9992 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpq7o0qtmv/model/ with prefix f9e324527fd34aef
[INFO 25-05-29 11:55:03.0102 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 25-05-29 11:55:03.0102 PDT kernel.cc:1061] Use fast generic engine


i:60


[WARNING 25-05-29 11:55:03.4307 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:03.4307 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:03.4307 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:03.8364 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpihqauwjb/model/ with prefix 61d87cb7fae84f91
[INFO 25-05-29 11:55:03.8397 PDT kernel.cc:1061] Use fast generic engine


i:61


[WARNING 25-05-29 11:55:04.2289 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:04.2289 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:04.2289 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:04.6378 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp7eop7zs_/model/ with prefix ddae26be26e240bf
[INFO 25-05-29 11:55:04.6436 PDT kernel.cc:1061] Use fast generic engine


i:62


[WARNING 25-05-29 11:55:04.9978 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:04.9978 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:04.9978 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:05.4295 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmptxafky3y/model/ with prefix 3f680cc28c5d4167
[INFO 25-05-29 11:55:05.4360 PDT kernel.cc:1061] Use fast generic engine


i:63


[WARNING 25-05-29 11:55:05.7969 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:05.7969 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:05.7969 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:06.1799 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpx27fqx_h/model/ with prefix c4d701d21a324599
[INFO 25-05-29 11:55:06.1851 PDT kernel.cc:1061] Use fast generic engine


i:64


[WARNING 25-05-29 11:55:06.5423 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:06.5423 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:06.5423 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:06.9524 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp0jq2jo24/model/ with prefix 339d8c48611b4b7e
[INFO 25-05-29 11:55:06.9582 PDT kernel.cc:1061] Use fast generic engine


i:65


[WARNING 25-05-29 11:55:07.3112 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:07.3112 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:07.3112 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:07.7601 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpx3h2sibr/model/ with prefix ebb7bfddc10147ee
[INFO 25-05-29 11:55:07.7674 PDT kernel.cc:1061] Use fast generic engine


i:66


[WARNING 25-05-29 11:55:08.1230 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:08.1231 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:08.1231 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:08.5080 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp35dc0c08/model/ with prefix 24c58a71a882413d
[INFO 25-05-29 11:55:08.5130 PDT kernel.cc:1061] Use fast generic engine


i:67


[WARNING 25-05-29 11:55:08.8926 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:08.8926 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:08.8926 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:09.3251 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpbvwdr6ur/model/ with prefix 987e1baec00a4ed3
[INFO 25-05-29 11:55:09.3313 PDT kernel.cc:1061] Use fast generic engine


i:68


[WARNING 25-05-29 11:55:09.7046 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:09.7046 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:09.7046 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:10.2720 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp36d8zapg/model/ with prefix 6789af0eb29e4491
[INFO 25-05-29 11:55:10.2847 PDT kernel.cc:1061] Use fast generic engine


i:69


[WARNING 25-05-29 11:55:10.6469 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:10.6469 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:10.6469 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:11.1250 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp90e48be6/model/ with prefix 1b02a73f2b094f3b
[INFO 25-05-29 11:55:11.1345 PDT kernel.cc:1061] Use fast generic engine


i:70


[WARNING 25-05-29 11:55:11.5343 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:11.5343 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:11.5343 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:11.9085 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpq6mtus3f/model/ with prefix 3f1fe91b8eea49da
[INFO 25-05-29 11:55:11.9129 PDT quick_scorer_extended.cc:911] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 25-05-29 11:55:11.9132 PDT kernel.cc:1061] Use fast generic engine


i:71


[WARNING 25-05-29 11:55:12.2795 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:12.2796 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:12.2796 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:12.6420 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpg43nhlrx/model/ with prefix dd7f0141fc994175
[INFO 25-05-29 11:55:12.6454 PDT kernel.cc:1061] Use fast generic engine


i:72


[WARNING 25-05-29 11:55:13.0067 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:13.0068 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:13.0068 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:13.4165 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp3uze7v29/model/ with prefix 104a73c590ed4939
[INFO 25-05-29 11:55:13.4225 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 25-05-29 11:55:13.4226 PDT kernel.cc:1061] Use fast generic engine


i:73


[WARNING 25-05-29 11:55:13.7815 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:13.7816 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:13.7816 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:14.1285 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpjo19h8nk/model/ with prefix fdc2d7447d65487d
[INFO 25-05-29 11:55:14.1318 PDT kernel.cc:1061] Use fast generic engine


i:74


[WARNING 25-05-29 11:55:14.5042 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:14.5042 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:14.5042 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:15.1702 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpy3ipjybn/model/ with prefix aac800172126444a
[INFO 25-05-29 11:55:15.1814 PDT kernel.cc:1061] Use fast generic engine


i:75


[WARNING 25-05-29 11:55:15.5544 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:15.5544 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:15.5544 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:16.2991 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpgmvq8833/model/ with prefix ce4f52c1834344d7
[INFO 25-05-29 11:55:16.3029 PDT kernel.cc:1061] Use fast generic engine


i:76


[WARNING 25-05-29 11:55:16.6750 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:16.6750 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:16.6750 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:17.0691 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp92p3dil8/model/ with prefix 8466f3823b4b4a0e
[INFO 25-05-29 11:55:17.0734 PDT kernel.cc:1061] Use fast generic engine


i:77


[WARNING 25-05-29 11:55:17.4447 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:17.4447 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:17.4447 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:17.8774 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpmtd8j1sk/model/ with prefix 7cabc57591d84fe2
[INFO 25-05-29 11:55:17.8839 PDT kernel.cc:1061] Use fast generic engine


i:78


[WARNING 25-05-29 11:55:18.2554 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:18.2554 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:18.2554 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:18.6242 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp9ymgr6w_/model/ with prefix cd436c17b5914294
[INFO 25-05-29 11:55:18.6269 PDT kernel.cc:1061] Use fast generic engine


i:79


[WARNING 25-05-29 11:55:19.0019 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:19.0019 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:19.0019 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:19.4765 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp2bkmpp1a/model/ with prefix 577fe47f5369498f
[INFO 25-05-29 11:55:19.4846 PDT kernel.cc:1061] Use fast generic engine


i:80


[WARNING 25-05-29 11:55:19.8530 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:19.8530 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:19.8530 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:20.2543 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpifaziaoj/model/ with prefix f9c7388eee7b4275
[INFO 25-05-29 11:55:20.2595 PDT kernel.cc:1061] Use fast generic engine


i:81


[WARNING 25-05-29 11:55:20.6340 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:20.6341 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:20.6341 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:21.0907 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpnbgi0cqg/model/ with prefix fe1943a448e04e6b
[INFO 25-05-29 11:55:21.0981 PDT kernel.cc:1061] Use fast generic engine


i:82


[WARNING 25-05-29 11:55:21.4647 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:21.4647 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:21.4647 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:21.9643 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmphidbok91/model/ with prefix fd0b85a2dea647a8
[INFO 25-05-29 11:55:21.9734 PDT kernel.cc:1061] Use fast generic engine


i:83


[WARNING 25-05-29 11:55:22.3599 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:22.3599 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:22.3599 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:22.8378 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpf1socrty/model/ with prefix 4b901b99d8e54ace
[INFO 25-05-29 11:55:22.8461 PDT kernel.cc:1061] Use fast generic engine


i:84


[WARNING 25-05-29 11:55:23.2144 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:23.2144 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:23.2144 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:23.6516 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp2rw2odwp/model/ with prefix c4e28a9c63c54892
[INFO 25-05-29 11:55:23.6584 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 25-05-29 11:55:23.6584 PDT kernel.cc:1061] Use fast generic engine


i:85


[WARNING 25-05-29 11:55:24.0224 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:24.0224 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:24.0225 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:24.4534 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmp_vcdjmn2/model/ with prefix 96c5f1b069444b48
[INFO 25-05-29 11:55:24.4603 PDT kernel.cc:1061] Use fast generic engine


i:86


[WARNING 25-05-29 11:55:24.8671 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:24.8671 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:24.8671 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:25.4090 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpm66_juz0/model/ with prefix a6d364c9136e4b5e
[INFO 25-05-29 11:55:25.4204 PDT kernel.cc:1061] Use fast generic engine


i:87


[WARNING 25-05-29 11:55:25.8152 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:25.8153 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:25.8153 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:26.2935 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpwypopdlq/model/ with prefix 270e05d011af406a
[INFO 25-05-29 11:55:26.3018 PDT kernel.cc:1061] Use fast generic engine


i:88


[WARNING 25-05-29 11:55:26.7104 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:26.7104 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:26.7104 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:27.1577 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpib1mhm3l/model/ with prefix 2341fc4930b74fa9
[INFO 25-05-29 11:55:27.1654 PDT kernel.cc:1061] Use fast generic engine


i:89


[WARNING 25-05-29 11:55:27.5326 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:27.5327 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:27.5327 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:27.9577 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpyd2u6fao/model/ with prefix 532f13dd27944aeb
[INFO 25-05-29 11:55:27.9633 PDT kernel.cc:1061] Use fast generic engine


i:90


[WARNING 25-05-29 11:55:28.3600 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:28.3601 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:28.3601 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:28.8167 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpjuhp8w7b/model/ with prefix 32eb555d848d4164
[INFO 25-05-29 11:55:28.8324 PDT kernel.cc:1061] Use fast generic engine


i:91


[WARNING 25-05-29 11:55:29.2321 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:29.2322 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:29.2322 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:29.7275 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpbj10cs_7/model/ with prefix ab37b7d241a8479e
[INFO 25-05-29 11:55:29.7359 PDT kernel.cc:1061] Use fast generic engine


i:92


[WARNING 25-05-29 11:55:30.1223 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:30.1223 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:30.1223 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:30.7955 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpwfe4c8ea/model/ with prefix f075dfaaf1a14949
[INFO 25-05-29 11:55:30.8103 PDT kernel.cc:1061] Use fast generic engine


i:93


[WARNING 25-05-29 11:55:31.1666 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:31.1666 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:31.1667 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:31.5374 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpr9_1zayf/model/ with prefix 596f238cb8cd45ae
[INFO 25-05-29 11:55:31.5414 PDT kernel.cc:1061] Use fast generic engine


i:94


[WARNING 25-05-29 11:55:31.9209 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:31.9209 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:31.9209 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:32.3333 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmphajgi3n4/model/ with prefix a7e6363165cc4f22
[INFO 25-05-29 11:55:32.3385 PDT kernel.cc:1061] Use fast generic engine


i:95


[WARNING 25-05-29 11:55:32.7120 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:32.7120 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:32.7120 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:33.1174 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpn18mebwb/model/ with prefix 0be9eb93c42940c6
[INFO 25-05-29 11:55:33.1231 PDT kernel.cc:1061] Use fast generic engine


i:96


[WARNING 25-05-29 11:55:33.4951 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:33.4951 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:33.4951 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:33.9320 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpzv2dt_k9/model/ with prefix e80f0ebadd4842eb
[INFO 25-05-29 11:55:33.9382 PDT abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 25-05-29 11:55:33.9383 PDT kernel.cc:1061] Use fast generic engine


i:97


[WARNING 25-05-29 11:55:34.2969 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:34.2969 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:34.2970 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:34.6866 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpld7d8gno/model/ with prefix ace65501b73f489e
[INFO 25-05-29 11:55:34.6903 PDT kernel.cc:1061] Use fast generic engine


i:98


[WARNING 25-05-29 11:55:35.0513 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:35.0513 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:35.0514 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:35.7015 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpsyjtuj7k/model/ with prefix c49de2b4b24b46c5
[INFO 25-05-29 11:55:35.7172 PDT kernel.cc:1061] Use fast generic engine


i:99


[WARNING 25-05-29 11:55:36.0826 PDT gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:36.0826 PDT gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 25-05-29 11:55:36.0826 PDT gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 25-05-29 11:55:36.8777 PDT kernel.cc:1233] Loading model from path /var/folders/4j/39rtwmqd5tg4g8dn1t6m037h0000gn/T/tmpuyb4ds2i/model/ with prefix aa64246986f04f5c
[INFO 25-05-29 11:55:36.8999 PDT kernel.cc:1061] Use fast generic engine


In [11]:
print(kaggle_predictions)

     PassengerId  Survived
0            892         0
1            893         1
2            894         0
3            895         0
4            896         1
..           ...       ...
413         1305         0
414         1306         1
415         1307         0
416         1308         0
417         1309         0

[418 rows x 2 columns]


# What is next

If you want to learn more about TensorFlow Decision Forests and its advanced features, you can follow the official documentation [here](https://www.tensorflow.org/decision_forests)